Ignore imports in ajava parsing (#357)
eisop/checker-framework: docs/CHANGELOG.md

try to find the closest non-artificial parent in the AST path to provide position
information.

> Made formatting rules for *.ajava files to be consistent with the ones for *.java
> files. Ignored imports when parsing ajava files.

Reviewer: ```suggestion
Formatting rules for `*.ajava` files are now consistent with the ones for `*.java` files.
Imports are now ignored when parsing `ajava` files.
```
